docs/source/referencing_data.rst

```diff
@@ -2,21 +2,48 @@
 Links to Existing Data
 =========================
 
+Metadata in the portal is connected by linking item types together.
+A simplified version of the data model is shown below, with items linking together to indicate how files were derived from experiments, samples, and donors.
+ 
+
+.. image:: /static/img/docs/data_model.png
+  :target: /static/img/docs/data_model.png
+  :alt: Data Model Image
+
+
+This allows for clear interpretation of the origin of sequence data.
 References to existing data are frequently required to submit new data. The links below will display all options available for referencing data already in the database.
 
+In the submission spreadsheet (See <a href="/docs/submission/getting-started-with-submissions", target="_blank">Getting Started with Submissions</a>), columns that are italicized indicate fields that should be unique identifiers for items of another type.
```

Comment:
Ah I see the issue. So there are a few different ways to handle in-line links in RST. You could do it inline like this (like the syntax used in the changelog):
```
In the submission spreadsheet (See `Getting Started with Submissions <docs/submission/getting-started-with-submissions>`_), columns that are italicized indicate fields that should be unique identifiers for items of another type.
```
or via reference:
```
.. _getting-started-with-submissions: docs/submission/getting-started-with-submissions

In the submission spreadsheet (See `Getting Started with Submissions`_), columns that are italicized indicate fields that should be unique identifiers for items of another type.
```

Unfortunately, RST doesn't have an equivalent to the `target="_blank"` functionality from HTML. So if you want to open the link in a new tab, you'll need to make that paragraph an HTML paragraph instead. That would look something like this:
```
.. raw:: html

<p>
In the submission spreadsheet (See <a href="/docs/submission/getting-started-with-submissions", target="_blank">Getting Started with Submissions</a>), columns that are italicized indicate fields that should be unique identifiers for items of another type.
</p>
```